### ISSUE TITLE
Multiple "extra" fields

### ISSUE THREAD

**Issue by nicolaheile** [SergeyMosin/Appointments#223]
Is it possible to extend the "extra field feature" to add multiple extra fields with JSON Code. This feature would be fantastic ;-)

**Comment by orcas-mgleich** [comment]
This would be nice, yeah

**Comment by spaetz** [comment]
Not sure if that solves your need, but in "Settings" -> Customize Public Page -> Advanced Setting -> Form designer (Beta) you can enter additional things. E.g. mine contains a (German):
{
  "tag": "textarea",
  "label": "Ihr Anliegen",
  "placeholder": "Bitte geben Sie Ihr Anliegen in Stichworten ein"
}
to have people write down a reason for their appointment. I think (but am not sure) you can have multiple fields. See 
https://github.com/SergeyMosin/Appointments/issues/24#issuecomment-721103321 for details on the form designer.

**Comment by LadySolveig** [comment]
@nicolaheile @orcas-mgleich @SergeyMosin  #259 Here is a temporarily workaround to support multiple custom fields without a GUI - only Json. Tested with all field types mentioned here #24 
Example for usage:
```
[
    {
      "tag": "select",
      "label": "Make your choice",
      "options":[
         "Cars",
         "Tanks",
         "Mermaids"
      ]
    } ,
  {
    "tag": "input",
    "type": "number",
    "label": "Enter number",
    "placeholder": "123456"
  } ,
  {
    "tag": "textarea",
    "label": "Enter text",
    "placeholder": "placeholder text"
  }
]
```

**Comment by CaptKrisp** [comment]
I am not able to specify multiple custom fields as @LadySolveig specifies.
Installed version: Appointments 1.11.7

**Comment by nicolaheile** [comment]
@CaptKrisp I created a Patch from the commit and patch the two files with the modification from @LadySolveig - you can simply seach for the modified lines and patch it by your own after each update from the Appointments app. The Line numbers does not match after the last update.

**Comment by CaptKrisp** [comment]
Thanks @SergeyMosin and @nicolaheile. I guess I didn't understand the #259 was a branch, as I'm not very adept at GitHub processes.
However, I had even considered diving into the code to add the functionality of multiple custom fields but have little extra time these days.

Thanks very much for the patch @nicolaheile, I'll put it to use! :)

**Comment by Taste-IT** [comment]
Hi,
i want to have multiple Fiels too. How to do that?

**Comment by iamkhalidbashir** [comment]
Why isnt this merged yet?

**Comment by iamkhalidbashir** [comment]
> @CaptKrisp I created a Patch from the commit and patch the two files with the modification from @LadySolveig - you can simply seach for the modified lines and patch it by your own after each update from the Appointments app. The Line numbers does not match after the last update.

I updated the file lines 
But I still cant see changes
do I need to restart AIO ?

**Comment by SergeyMosin** [comment]
available in v1.14.6
